apps/web-connect/src/features/checkout/components/ChooseQuantityRow.tsx

```diff
@@ -16,13 +16,6 @@ export function ChooseQuantityRow(): JSX.Element {
     // Destructure necessary values from the WebBuyKeysContext
     const { quantity, setQuantity, maxSupply } = useWebBuyKeysContext();
 
-    const calculateMaxPurchase = (): number => {
-        if (quantity > maxSupply) {
-            return maxSupply;
-        }
-        return quantity
-    }
-
```

Comment:
Why is this not needed anymore ?

Reply:
This was never needed, it was added in error and was part of the bug.